packages/@angular/cli/commands/completion.ts

```diff
@@ -117,14 +117,15 @@ const CompletionCommand = Command.extend({
         });
       }
 
-      let opts = '';
+      let opts: string[] = [];
       if (command.blueprints && command.blueprints[0]) {
-        opts += extractBlueprints(command.blueprints);
+        opts = opts.concat(extractBlueprints(command.blueprints));
       }
 
       if (command.availableOptions && command.availableOptions[0]) {
-        opts += extractOptions(command.availableOptions);
-        caseBlock = caseBlock + '    ' + com.sort().join('|') + ') opts="' + opts + '" ;;\n';
+        opts = opts.concat(extractOptions(command.availableOptions));
+        const optsStr = opts.sort().join(' ');
+        caseBlock = caseBlock + '    ' + com.sort().join('|') + ') opts="' + optsStr + '" ;;\n';
```

Comment:
would be nicer wit a template string ` ${caseBlock}    ${ com.sort().join('|') } ) opts= ${optsStr} ;;\n`   , it is typescript after all, isnt the starting left parenthesis missing as well?